Chat-Commands.md

```diff
@@ -185,6 +185,10 @@ For some miscellaneous activities, the following shorthand versions can be used:
 `hunterrumour`, `hunter contract`, `hunter contracts`, `hunter tasks`, `hunter task`, `rumours`, `rumour` -> `Hunter Rumours`  
 `herbi` -> `Herbiboar`  
 `bird egg`, `bird eggs`, `bird's egg`, `bird's eggs` -> `Bird's egg sacrifices`
+`crystal chest` -> `crystal chest`
+`larran small chest`, `larran's small chest` -> `Larran's small chest`
+`larran chest`, `larran's chest`, `larran big chest`, `larran's big chest` -> `Larran's big chest`
+`brimstone chest` -> `Brimstone chest`
```

Comment:
```suggestion
`bird egg`, `bird eggs`, `bird's egg`, `bird's eggs` -> `Bird's egg sacrifices`  
`larran chest`, `larran's chest`, `larran big chest`, `larran's big chest` -> `Larran's big chest`  
`larran small chest`, `larran's small chest` -> `Larran's small chest`  
```

- Adds missing line breaks
- Switches order for big/small chest because likely nobody cares about the small chest (nitpick)
- Removes the full names -> I don't think there's a reason to add the full names in the list of abbreviations? Feel free to correct me, but I don't think we do this anywhere.

Reply:
Suggestion applied, thanks for proof reading! :)